.changeset/smooth-crabs-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/backend': minor
+---
+
+The `clockSkewInSeconds` property is now deprecated from the `verifyJWT` options in favour of the new `clockSkewInMs` property. The old value incorrectly accepted a value in milliseconds so the behaviour will remain the same.
```

Comment:
❓ This could be misunderstood. The problem was just the name of the property. Let's say something like:

```
The old property accepted a value in milliseconds, so this change fixes the property name.
```